### PR TITLE
Add migration according to Node change & Release 0.22.0

### LINKED ISSUE #157: Add migration according to `Node` change

### PR DESCRIPTION
**Pull request by sophie-cluml**
closes #157

### CODE REVIEW

**Comment by sophie-cluml** [comment]
Hello @sehkone and @dayeon5470, this PR is for https://github.com/aicers/aice-web/pull/429. Your inputs are very welcome.

**Comment by sehkone** [comment]
@minshao @msk,

This migration is needed because of [this](https://github.com/aicers/review-web/pull/95).

File: src/collections/indexed_map.rs [hunk 1]
```diff
@@ -60,6 +63,41 @@ impl<'a> IndexedMap<'a> {
         }
         self.db.get_cf(self.cf, key).context("cannot read entry")
     }
+
+    /// Updates a key's value from an old one to a new one.
+    ///
+    /// # Errors
+    ///
+    /// Returns an error if the key does not exist, or the database operation fails.
+    pub fn update_by_key(&self, key: &[u8], new_value: &[u8]) -> Result<()> {
```
Comment (sehkone) [inline src/collections/indexed_map.rs:72]: I think the current value should be checked if it is the same as what the user is trying to update.

Comment (sophie-cluml) [reply src/collections/indexed_map.rs]: I found`Indexed.overwrite` does what I wan to do, so I changed the migration to use it.

**Comment by minshao** [comment]
For clarity and consistency, please create a separate pull request for the version bump next time.

**Review by minshao — APPROVED** [review]

**Comment by msk** [comment]
Manually resolved conflicts and pushed to main, because I was not allowed to modify the original PR in the forked repo.

For future reference, it's generally a good practice to separate functional changes from version updates. This separation helps in streamlining the review process and avoids potential conflicts with other ongoing work intended for the same version release.

**Comment by sophie-cluml** [comment]
Well noted and thanks for handling it! I will close this PR.

**Pull request closed without merging**